content/en/docs/contributing/localization.md

```diff
@@ -93,6 +94,31 @@ path would become `/ja/docs/some-page` when rendered from a Japanese page.
 
 {{% /alert %}}
 
+### Link definition labels {#link-labels}
+
+Do **not** translate [labels] of markdown [link definitions][]. Instead, rewrite
+the label as translated link text. For example, consider the following markdown:
+
+```markdown
+[Hello], world! Welcome to the [OTel website][].
+
+[hello]: https://code.org/helloworld
+[OTel website]: https://opentelementry.io
+```
+
+This would be translated in French as:
+
+```markdown
+[Bonjour][hello], le monde! Bienvenue sur le [site OTel][OTel website].
+
+[hello]: https://code.org/helloworld
```

Comment:
And if capitalization matters, should these `hello` references also be capitalized to match the English version?